src/Upgrade/Infrastructure/PackageManager/CommandExecutor/ComposerCommandExecutor.php

```diff
@@ -199,6 +209,49 @@ public function update(): PackageManagerResponseDto
         )));
     }
 
+    /**
+     * @return \Upgrade\Application\Dto\PackageManagerResponseDto
+     */
+    public function updateLockHash(): PackageManagerResponseDto
+    {
+        $package = $this->getFirstAvailablePackage();
+
+        $command = [
+            ...explode(' ', static::UPDATE_COMMAND_NAME),
+            sprintf('%s:%s', $package['name'], $package['version']),
+            static::NO_PLUGINS_FLAG,
+            static::NO_SCRIPTS_FLAG,
+            static::NO_INSTALL_FLAG,
```

Comment:
Have you checked that the version in lock state the same?

Reply:
I get this version from the lock file